evalai/challenges.py

```diff
@@ -24,6 +24,16 @@ class Challenge(object):
     Stores user input ID's.
```

Comment:
Please remove `full stop` from all the sentences. It looks odd that we have it in some places & some places are without it.